.travis.yml

```diff
@@ -16,6 +18,15 @@ python:
 - '3.6'
 matrix:
   include:
+  - os: osx
+    language: generic
+    env: NMODE=nikola
+    python: '3.6'
+    before_install:
+    - brew upgrade pyenv
```

Comment:
As an aside, I was also not a fan of pyenv for quite a while but pyenv-virtualenv combined with `pyenv local` has really brought me around.